webdriver/support/fixtures.py

```diff
@@ -115,3 +117,83 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def switch_to_inactive(session):
+    """Create a new browsing context, switch to that new context, and then
+    delete the context."""
+    def switch_to_inactive():
+        initial = session.send_session_command("GET", "window/handles")
+
+        session.send_session_command("POST",
+                                     "execute/sync",
+                                     dict(script="window.open();", args=[]))
+        with_new = session.send_session_command("GET", "window/handles")
+
+        assert len(initial) == len(with_new) - 1
+
+        new_handle = (set(with_new) - set(initial)).pop()
```

Comment:
This set has exactly one element--see the preceding `assert` statement. That
might make using `pop` a little misleading, though. If you'd like, I can update
the existing code to simply reference the first (and only) member via `[0]`.